Tests/ChainsafeRPCTests.cs

```diff
@@ -197,24 +197,6 @@ public void GetSignerTest()
             Assert.AreEqual(1337, signer.GetChainId().Result);
         }
 
-        [Test]
-        public void SendTransactionTest()
```

Comment:
we already had a send transaction test, so I moved to the new test class and improve this test base on ethers.js test.

Also, the idea is not move the tests located in ChainsafeRPCTests to different test classes depending on the functionality